### ISSUE TITLE
[simple-social-icons] shortcode

### ISSUE THREAD

**Issue by nickcernis** [copyblogger/simple-social-icons#50]
We've seen repeated requests in the WP.org forums for a Simple Social Icons shortcode to output icons within a post.

https://wordpress.org/support/topic/shortcode-477/

May require the addition of a global settings page like we have in Simple Share.

**Comment by marksabbath** [comment]
Hey @nickcernis since this is a widget, what do you think about using an approach like `amr-shortcode-any-widget`plugin? Is there any suggestion in your mind related to this?

**Comment by nickcernis** [comment]
@marksabbath How do you feel about putting this issue into [backlog](https://github.com/studiopress/simple-social-icons/milestone/1) for now?

It looks like a dedicated shortcode would be a bigger project (requires plugin restructuring and a custom admin page). I'm not sure the utility a shortcode offers is worth it right now, and think a Gutenberg block may be more useful once that becomes available as an option to us.

If the amr-shortcode-any-widget plugin works as an alternative, we could suggest that as a workaround for now for those asking.

cc @mattlawck @dreamwhisper

**Comment by marksabbath** [comment]
Sounds like a reliable plan @nickcernis and I do agree that maybe, a Gutenberg block would be way more useful than a shortcode at this point.

Let's wait for @dreamwhisper and @mattlawck thoughts :)

Thanks for your help man!

**Comment by mattlawck** [comment]
I agree with that, if its a large feature that doesn't yet exist, should probably be the focus of its own version?

**Comment by marksabbath** [comment]
Yeap, maybe a version focused on this feature would make more sense. Moving it to backlog. Thanks guys!

**Comment by cinghaman** [comment]
Since Gutenberg block has been there for a while, any chance for adding social icons to Gutenberg block?
